tests/pact/index.pact.spec.js

```diff
@@ -0,0 +1,51 @@
+'use strict'
+
+const { pactWith } = require('jest-pact')
+const { Hello_Greet } = require('../../ui/client/hello/index')
+
+pactWith({ consumer: 'OCIS-Hello_UI', provider: 'OCIS-Hello_API' }, provider => {
+  describe('Hello API', () => {
```

Comment:
Codacy found an issue: ['describe' is not defined.](https://app.codacy.com/gh/owncloud/ocis-hello/pullRequest?prid=5869027)